traversal/planner/GraphPlanner.java

```diff
@@ -217,135 +160,102 @@ private PlannerVertex.Type typeVertex(StructureVertex.Type structureVertex) {
     }
 
     private void initialise() {
-        initialiseVariables();
-        initialiseConstraintsForVariables();
-        initialiseConstraintsForEdges();
+        createVariables();
+        createConstraints();
```

Comment:
To connec the storyline even more fully we could call these `createOptimiserVariables()` and `createOptimiserConstraints()`

Then we've definitely set on the vocabulary:
1. `Planner` objects are our own objects 
2. `Optimiser` is the linear optimiser
3. we `register` planner vertices/edges, then `create` optimiser variables and constraints
4. finally, we greedily `initialise` the `Optimiser` variables with a valid solution.

by that logic `setInitialValues` should become `initialiseOptimiserValues` and `PlannerVertex/Edge.initialise()` should become`initialiseOptimiserValues()` as well!